### PR TITLE
minor tweaks,optimize code

### PR DESCRIPTION
**Pull request by lancoLiu**

### CODE REVIEW

**Review by zihengCat — APPROVED** [review]
LGTM

**Review by daheige — APPROVED** [review]
@appleboy support it!

File: render/json.go [hunk 1]
```diff
@@ -102,8 +102,7 @@ func (r SecureJSON) Render(w http.ResponseWriter) error {
 	// if the jsonBytes is array values
 	if bytes.HasPrefix(jsonBytes, bytesconv.StringToBytes("[")) && bytes.HasSuffix(jsonBytes,
 		bytesconv.StringToBytes("]")) {
-		_, err = w.Write(bytesconv.StringToBytes(r.Prefix))
-		if err != nil {
+		if _, err = w.Write(bytesconv.StringToBytes(r.Prefix)); err != nil {
```
Comment (kszafran) [inline render/json.go:105]: I'd even write:
```suggestion
		if _, err := w.Write(bytesconv.StringToBytes(r.Prefix)); err != nil {
```
here and elsewhere. This way you don't reassign to the outer `err` variable.

**Review by thinkerou — APPROVED** [review]
lgtm

**Review by appleboy — APPROVED** [review]

**Pull request merged**